maker/src/position.rs

```diff
@@ -1,30 +1,109 @@
+use anyhow::Result;
 use async_trait::async_trait;
+use hedging::derive_hedging_action;
+use rust_decimal::prelude::ToPrimitive;
 use rust_decimal::Decimal;
+use rust_decimal::RoundingStrategy;
 use serde::Serialize;
 use std::collections::HashMap;
 use std::collections::HashSet;
 use std::fmt::Display;
 use std::hash::Hash;
 use std::ops::Add;
+use std::time::Duration;
 use uuid::Uuid;
+use xtra::Mailbox;
 
+mod bitmex;
+mod hedging;
 mod tentenone;
 
 pub struct Manager {
     position: Position,
+    bitmex_http_client: bitmex_client::client::Client,
+}
+
+#[async_trait]
+impl xtra::Actor for Manager {
+    type Stop = ();
+
+    async fn started(&mut self, mailbox: &mut Mailbox<Self>) -> Result<(), Self::Stop> {
+        tokio::spawn({
+            let mailbox = mailbox.clone();
+            async move {
+                loop {
+                    // We sleep first to allow the 10101 and BitMEX positions to be up-to-date
+                    // before we start hedging.
+                    tokio::time::sleep(Duration::from_secs(60)).await;
```

Comment:
🤔 Is this safe? what if the BitMEX positions are not up-to-date?

Reply:
One thing we could do is to only hedge if we know that the positions are up-to-date by at least a minute.

Reply:
Shouldn't the first hedge event after we have synchronised the tentenone position?